docker-cloud/cloud-swarm/create-cloud-swarm.md

```diff
@@ -4,10 +4,7 @@ keywords: swarm mode, swarms, create swarm, Cloud
 title: Create a new swarm in Docker Cloud
 ---
 
-<b>Note</b>: All Swarm management features in Docker Cloud are free while in Beta.
-{: style="text-align:center" }
-
---------
+> **Note:** All Swarm management features in Docker Cloud are free while in Beta.
```

Comment:
Let's move "free while in beta" reference to the "Using swarm mode with Docker Cloud" doc (so as to qualify that all swarm mode functionality is free while in beta).